_data/data.yml

```diff
@@ -1013,37 +1086,55 @@ publications:
   ### DO NOT FORGET TO ADD YOUR NEW TAG &XXXXXX TO THE ENTRIES AT THE BOTTOM ###
   ### IF YOU FORGET, YOUR ENTRIES WILL NOT SHOW UP ON THE SITE! ###
 
+    - &almashaqbeh_microcash_fc20
+      anchor: almashaqbeh_microcash_fc20
+      title: "Microcash: Practical Concurrent Processing of Micropayments"
+      authors:
+        - name: "G. Almashaqbeh, A. Bishop, J. Cappos"
+      project: *cachecash
+      booktitle: ""
+      year: ""
+      pages: ""
+      publisher: ""
+      link: "/papers/almashaqbeh_microcash_fc20"
+      abstract: "Micropayments have a large number of potential applications. However, processing these small payments individually can be expensive, with transaction fees often exceeding the payment value itself. By aggregating the small transactions into a few larger ones, and using cryptocurrencies, today’s decentralized probabilistic micropayment schemes can reduce these fees. Unfortunately, existing solutions force micropayments to be issued sequentially, thus to support fast issuance rates a customer needs a large number of escrows, which bloats the blockchain. Moreover, these schemes incur a large computation and bandwidth overhead, limiting their applicability in large-scale systems. In this paper, we propose MicroCash, the first decentralized probabilistic framework that supports concurrent micropayments. MicroCash introduces a novel escrow setup that enables a customer to concurrently issue payment tickets at a fast rate using a single escrow. MicroCash is also cost effective because it allows for ticket exchange using only one round of communication, and it aggregates the micropayments using a non-interactive lottery protocol that requires only secure hashing and supports fixed winning rates. Our experiments show that MicroCash can process thousands of tickets per second, which is around 1.7-4.2x times the rate of a state-of-the-art sequential micropayment system. Moreover, MicroCash supports any ticket issue rate over any period using only one escrow, while the sequential scheme would need more than 1000 escrows per second to permit high rates. This enables our system to further reduce transaction fees and data on the blockchain by ∼ 50%."
+
+
     - &moore_crashsim_issre_19
       anchor: moore_crashsim_issre_19
       title: "Charting a Course Through Uncertain Environments: SEA Uses Past Problems to Avoid Future Failures"
       authors:
-        - name: "P. Moore, J. Cappos, P. Frankl,  and T. Wies"
+        - name: "P. Moore, J. Cappos, P. Frankl, and T. Wies"
       project: *crashsimulator
-      booktitle: "to be presented at the 30th IEEE International Symposium on Software Reliability Engineering (ISSRE'19)"
+      booktitle: "30th IEEE International Symposium on Software Reliability Engineering (ISSRE'19) <br/> Best Paper Award"
```

Comment:
Is using angle brackets allowed in yml files?

Reply:
To be honest, I'm not sure. I noticed this was how the "best paper" was highlighted in Dan's paper. I can remove it. Is this why checks failed.

Reply:
```suggestion
      booktitle: "30th IEEE International Symposium on Software Reliability Engineering (ISSRE'19) Best Paper Award"
```